beta/src/components/MDX/ExpandableExample.tsx

```diff
@@ -8,23 +8,21 @@ import {IconChevron} from '../Icon/IconChevron';
 import {IconDeepDive} from '../Icon/IconDeepDive';
 import {IconCodeBlock} from '../Icon/IconCodeBlock';
 import {Button} from '../Button';
+import {H4} from './Heading';
 
 interface ExpandableExampleProps {
   children: React.ReactNode;
-  title: string;
   excerpt?: string;
   type: 'DeepDive' | 'Example';
 }
 
-function ExpandableExample({
-  children,
-  title,
-  excerpt,
-  type,
-}: ExpandableExampleProps) {
+function ExpandableExample({children, excerpt, type}: ExpandableExampleProps) {
   const [isExpanded, setIsExpanded] = React.useState(false);
   const isDeepDive = type === 'DeepDive';
   const isExample = type === 'Example';
+  const hasTitle =
+    Array.isArray(children) &&
+    ['h2', 'h3', 'h4'].includes(children[0].type.mdxName);
```

Comment:
> nit: Does type always exists in the `children[0]`? If not we could optional chain else chill 👍🏻

Yeah, I think it always exists, according to its type definition. No matter how, since the first child should always be the title, that should be ok, and the case with an optional title was also been tested.

Reply:
@harish-sethuraman  thx! updated!